doozerlib/rhcos.py

```diff
@@ -234,6 +235,13 @@ def __init__(self, runtime, pullspec_for_tag: Dict[str, str], brew_arch: str):
             if self.build_id and self.build_id != build_id:
                 raise Exception(f'Found divergent RHCOS build_id for {pullspec_for_tag}. {build_id} versus {self.build_id}')
             self.build_id = build_id
+            try:
+                openshift_os_commit = str(image_info.config.config.Labels['com.coreos.redhat-coreos-commit'])
+            except AttributeError:
+                raise Exception(f'Unable to determine RHCOS openshift/os commit from tag {tag} pullspec {pullspec}. Retrieved image info: {image_info_str}')
+            if self.openshift_os_commit and self.openshift_os_commit != openshift_os_commit:
+                raise Exception(f'Inconsistent RHCOS openshift/os commits for {pullspec_for_tag}')
```

Comment:
Would like to have the different commits in the error message

Reply:
This is how the error message looks like:
```
assembly_issues:
  rhcos:
  - code: INCONSISTENT_OS_COMMIT
    msg: 'Found diiferent openshift/os commits in builds [RHCOSBuild:x86_64:411.86.202208291737-0,
      RHCOSBuild:ppc64le:411.86.202208291739-0, RHCOSBuild:s390x:411.86.202208311151-0,
      RHCOSBuild:aarch64:411.86.202208310958-0] (private=False): {''765d2beb337b80b7fdad356f79abe54d7fdfd5c3'':
      {''ppc64le'', ''x86_64''}, ''0f001449a6690beff3aede4aa775a7e8d09f302f'': {''s390x'',
      ''aarch64''}}'
```